ethereum/contracts/ETHApp.sol

```diff
@@ -88,6 +93,9 @@ contract ETHApp is RewardController, AccessControl {
         // revert in case of overflow.
         uint128 value = (msg.value).toUint128();
 
+        (bool success, ) = address(vault).call{value: msg.value}("");
```

Comment:
what's the intention here? i guess to check that the vault is working? but value will probably be 0 in the constructor tx so seems kinda weird?